malaria-ios/ViewControllers/UserProfilePillTableViewCell.swift

```diff
@@ -0,0 +1,45 @@
+import UIKit
+import DoneToolbarSwift
+
+// MARK: - Table View Cell
+
+class UserProfilePillTableViewCell: UITableViewCell {
+  
+  @IBOutlet weak var pillName: UILabel!
+  @IBOutlet weak var quantityTextField: UITextField!
+  
+  weak var delegate : SaveRemainingPillsProtocol?
+  
+  var toolBar: ToolbarWithDone!
+  
+  func updateCellWithParameters(delegate: SaveRemainingPillsProtocol,
```

Comment:
I understand your concern. I usually put between 80 and 100ish. But I don't
follow it religiously... depends on the case.

2016-06-25 20:52 GMT+01:00 Teodor Ciuraru <notifications@github.com>:

> In malaria-ios/ViewControllers/UserProfilePillTableViewCell.swift
> <https://github.com/systers/malaria-app-ios/pull/52#discussion_r68493112>:
>
> > @@ -0,0 +1,45 @@
> > +import UIKit
> > +import DoneToolbarSwift
> > +
> > +// MARK: - Table View Cell
> > +
> > +class UserProfilePillTableViewCell: UITableViewCell {
> > +
> > +  @IBOutlet weak var pillName: UILabel!
> > +  @IBOutlet weak var quantityTextField: UITextField!
> > +
> > +  weak var delegate : SaveRemainingPillsProtocol?
> > +
> > +  var toolBar: ToolbarWithDone!
> > +
> > +  func updateCellWithParameters(delegate: SaveRemainingPillsProtocol,
>
> I'm trying to keep lines under 80 characters, but I'm not sure if it's
> necessary anymore (I will check the coding styles again).
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/systers/malaria-app-ios/pull/52/files/1bd98847e91e9af59711c1205e2643b2daf28fd7#r68493112>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEgjsT0Ts5tB_yKzdps-7-51lYkV7r9kks5qPYbwgaJpZM4I-I45>
> .
>



-- 
Os melhores cumprimentos,
Bruno Henriques